packages/desktop-gui/cypress/integration/project_spec.coffee

```diff
@@ -141,6 +142,90 @@ describe "Project", ->
           expect(@ipc.closeProject).to.be.called
           expect(@ipc.openProject).to.be.called
 
+    it "renders markdown", ->
+      @withMarkdown = '**this** is _markdown_'
+      @markdownErr = {
+        name: "Error",
+        message: @withMarkdown,
+        stack: "[object Object]↵"
```

Comment:
when we see `[object Object]` like this it generally indicates that an object was `toString()` instead of being properly serialized via `JSON.stringify(...)` so it's odd to see it here in a test, since it shouldn't actually ever come over as this.

Reply:
I copied [this error object](https://github.com/cypress-io/cypress/blob/develop/packages/desktop-gui/cypress/integration/project_spec.coffee#L86) and shortened it, but I'll change it.